FinTech.Logic/BusinessLogic/Account/AccountBL.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FinTech.Data;
+using System;
+
+namespace FinTech.Logic
+{
+    public abstract class AccountBL
+    {
+        public  Account CreateAccount(User user)
```

Comment:
Check method name in your design.

Need to pass the UserDto not User (applicable for all classes)

Reply:
Done